@here/harp-materials/lib/MapMeshMaterials.ts

```diff
@@ -1172,4 +1198,3 @@ applyMixinsWithoutProperties(MapMeshStandardMaterial, [FadingFeatureMixin]);
 applyMixinsWithoutProperties(MapMeshBasicMaterial, [ExtrusionFeatureMixin]);
 applyMixinsWithoutProperties(MapMeshStandardMaterial, [ExtrusionFeatureMixin]);
 applyMixinsWithoutProperties(MapMeshBasicMaterial, [DisplacementFeatureMixin]);
-applyMixinsWithoutProperties(MapMeshStandardMaterial, [DisplacementFeatureMixin]);
```

Comment:
we were applying the DisplacementFeatureMixin to a material that already has a displacement map.

Reply:
Wright... I did not spotted it before :+1: .